### DISCUSSION TITLE
There's no backup section in the settings

### DISCUSSION THREAD

**Discussion by K4LCIFER** [nextcloud/all-in-one#4375] [Questions] [answered]
### Steps to reproduce
1. Unknown. I just set up the AIO container.

### Expected behavior

The docs reference a backup section, so I would expect that there is a backup section in the settings.

### Actual behavior

I have no backup section.

### Host OS 

Ubuntu Server

#### Nextcloud AIO version 

v7.13.1

#### Current channel

latest

#### Other valuable info 

No backup section as seen in the following image:

![image](https://github.com/nextcloud/all-in-one/assets/7829231/d58680bb-161c-4524-ad54-e411995a1e2d)

Nextcloud Hub 7 v28.0.3

**Comment by EpicJosch** [comment, marked as answer]
Do you talk about the backup section from AIO (Borg Backup)? I don't think that there is a NC App for this and the section is only visible in the AIO Container web panel

**Reply by K4LCIFER** [reply]
Ah, yeah. I was under the impression that it would be an app.
